distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Send_Response.enso

```diff
@@ -0,0 +1,33 @@
+import project.Data.Text.Text
+import project.Network.Email.Email
+from project.Visualization.Table_Viz_Data import Table_Viz_Data, Table_Viz_Header
+
+## Response from sending an email.
+type Send_Response
+    ## No email was sent, this was just a test. Used only by the Test_Only Email_Provider.
+    Test email:Email
+
+    ## No email was sent, workflow was run in dry-run mode.
+    Dry_Run email:Email
+
+    ## Email was sent successfully.
+    Sent email:Email
+
+    ## Email sending failed.
+    Failed email:Email
+
+    ## ---
+       private: true
+       ---
+    to_text self -> Text =
+        case self of
+            Send_Response.Test _ -> "Test Only"
+            Send_Response.Dry_Run _ -> "Dry Run"
+            Send_Response.Sent _ -> "Sent"
+            Send_Response.Failed _ -> "Failed"
+
+## ---
+   private: true
+   ---
+Table_Viz_Data.from (that:Send_Response) = 
```

Comment:
- `Standard.Base` is referencing `Standard.Visualization`!
- e.g. creating a cyclic dependency between the libraries
   - I am surprised that it even works
- anyway having a cyclic dependency isn't good idea
- please **avoid** `Standard.Base` to reference any other libraries

Reply:
We have a ticket to think through the Standrd.Vis work - this is the current solution we can consider restructuring when time allows.